roles/static_inventory/tasks/checkpoint.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: create static inventory dir
+  file:
+    path: "{{ inventory_path }}"
```

Comment:
This would require manual steps, like creating a *right* directory (or the guide will stop working for users with custom inventory paths) and strictly following the guide's described manual steps, which only brings more chaos. I think ansible must take care of it instead, no manual/external configuration steps.